### PR TITLE
feat: Added chapter 5 challenge 2 part 5

### PR DESCRIPTION
**Pull request by daviroo**
## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

**Comment by daviroo** [comment]
@benalleng I've had a few issues with this:
- I can't get the javascript solution to work.  I'm having trouble parsing the hex values.  They seem to be getting truncated somehow and I can't work out how yet (was hoping you might have seen this before).
- How do I link the help page to the content?

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -879,6 +879,31 @@ const translations = {
...
       title: 'Verify the signature',
     },
+    verify_signature_five: {
+      title: 'Verify the signature',
+      heading: 'Verify the signature!',
+      success: "You've done it! You've verified Satoshi's signature!",
+      paragraph_one:
+        'At this point we have everything we need to do some ECDSA math.',
+      paragraph_two:
+        'The ECDSA signature verification algorithm is explained <Link className="underline" href="https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Signature_verification_algorithm" target="_blank">here</Link> and <Link className="underline" href="https://www.secg.org/sec1-v2.pdf" target="_blank">here</Link> (Go to page 46, section 4.1.4).',
```
Comment (benalleng) [inline i18n/locales/en.ts:889]: We might just want to add a page number fragement here to link to 4.1.4 in the pdf `#page=52`

**Comment by benalleng** [comment]
Closes #717

**Review by benalleng — APPROVED** [review]
LGTM!

**Pull request merged**